README.rst

```diff
@@ -225,6 +225,12 @@ response will contain an error object with a ``debug_url``:
 
 This tells you to find the traceback interpreter at ``<host>:<port>/debug/1234567890``.
 
+Logging
+-------
+
+The registry has a default logger, ``Registry._logger`` and it logs all errors that are not defined
+by ``typedjsonrpc``. You can configure the logger in ``Registry._get_default_logger()``.
```

Comment:
You can actually say that you can access the logger by doing:

    import logging
    logger = logging.getLogger("typedjsonrpc.registry")
    # Do configuration to this logger